test/browser/sampleTest.ts

```diff
@@ -16,8 +17,10 @@ describe('sampleTest', () => {
   it('should respond to changing obsArray', async function() {
     assert.deepEqual(await getText(driver.findAll('#out1 li')), []);
 
-    await driver.find('#in1 input').sendKeys("foo", Key.ENTER);
-    assert.deepEqual(await getText(driver.findAll('#out1 li')), ["+foo"]);
+    await waitToPass(async () => {
```

Comment:
It feels like this might mask a problem (e.g. if typing into the input doesn't always work), but it's not actually an important test of functionality, so it doesn't matter. I would just ask for a comment saying that it's a hack because the test fails for some mysterious reason, but that it doesn't matter here so it's OK to sweep under the rug.